### PR TITLE
Provide id for post actions

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Add an `id` to each Post Action

#### Ticket Link
https://github.com/mattermost/mattermost-server/pull/13225

### CODE REVIEW

File: source/developer/interactive-messages.rst [hunk 1]
```diff
@@ -200,6 +205,9 @@ Parameters
 
 Below is a brief description of each parameter to help you customize the interactive message button and menu in Mattermost. For more information on message attachments, `see our documentation <https://docs.mattermost.com/developer/message-attachments.html>`__.
 
+ID
+  A per post unique identifier.
```
Comment (justinegeffen) [inline source/developer/interactive-messages.rst:209]:
```suggestion
  A unique identifier required for each post.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Just one suggestion, not a blocker. Thanks a lot for putting this together, really appreciate it. :)

File: source/developer/interactive-messages.rst (hunk 1 above)
Comment (hanzei) [reply source/developer/interactive-messages.rst:209]: @justinegeffen That is actually not what I meant. The ID is unique per post. It's not required for every post action. How can I make this more clear?

Comment (justinegeffen) [reply source/developer/interactive-messages.rst:209]: I think we can go with what you've got. No need to over-complicate it - the wording does make sense.

**Review by levb — APPROVED** [review]

**Pull request merged**